spec/namespaces/security_analytics.yaml

```
@@ -0,0 +1,238 @@
+        url: https://docs.opensearch.org/docs/latest/security-analytics/api-tools/correlation-eng/#list-correlations-for-a-finding-belonging-to-a-log-type
+      parameters:
+        - $ref: '#/components/parameters/security_analytics.search_finding_correlations::query.detector_type'
+        - $ref: '#/components/parameters/security_analytics.search_finding_correlations::query.finding'
+        - $ref: '#/components/parameters/security_analytics.search_finding_correlations::query.nearby_findings'
+        - $ref: '#/components/parameters/security_analytics.search_finding_correlations::query.time_window'
+      responses:
+        '200':
+          $ref: '#/components/responses/security_analytics.search_finding_correlations@200'
+
+components:
+  responses:
+    security_analytics.get_alerts@200:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/security_analytics.alerts.yaml#/components/schemas/GetAlertsResponse'
+    security_analytics.get_findings@200:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/security_analytics.findings.yaml#/components/schemas/GetFindingsResponse'
+    security_analytics.search_finding_correlations@200:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/security_analytics.findings.yaml#/components/schemas/SearchFindingCorrelationsResponse'
+
+  parameters:
+    security_analytics.get_alerts::query.detector_id:
+      name: detector_id
+      in: query
+      description: The ID of the detector used to fetch alerts. Optional when detectorType is specified. Otherwise required.
+      schema:
+        type: string
+    security_analytics.get_alerts::query.detectorType:
+      name: detectorType
+      in: query
+      description: The type of detector used to fetch alerts. Optional when detector_id is specified. Otherwise required.
+      schema:
+        type: string
+    security_analytics.get_alerts::query.severityLevel:
+      name: severityLevel
+      in: query
+      description: Used to filter by alert severity level. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.get_alerts::query.alertState:
+      name: alertState
+      in: query
+      description: Used to filter by alert state. Optional.
+      schema:
+        type: string
+        enum: [ACKNOWLEDGED,ACTIVE, COMPLETED, DELETED, ERROR]
+      required: false
+    security_analytics.get_alerts::query.sortString:
+      name: sortString
+      in: query
+      description: Specifies which string Security Analytics uses to sort the alerts. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.get_alerts::query.sortOrder:
+      name: sortOrder
+      in: query
+      description: The order used to sort the list of findings. Possible values are asc or desc. Optional.
+      schema:
+        type: string
+        enum: [asc, desc]
```

This can re-use the existing enum:
```suggestion
      schema:
        $ref: '../schemas/_common.yaml#/components/schemas/SortOrder'
```